fe/fe-core/src/main/java/org/apache/doris/load/StreamLoadRecordMgr.java

```diff
@@ -0,0 +1,112 @@
+                            LOG.info("receive stream load record info from backend: {}. label: {}, db: {}, tbl: {}, user: {}, user_ip: {}," +
+                                            " status: {}, message: {}, error_url: {}, total_rows: {}, loaded_rows: {}, filtered_rows: {}," +
+                                            " unselected_rows: {}, load_bytes: {}, start_time: {}, finish_time: {}.",
+                                    backend.getHost(), streamLoadItem.getLabel(), streamLoadItem.getDb(), streamLoadItem.getTbl(), streamLoadItem.getUser(), streamLoadItem.getUserIp(),
+                                    streamLoadItem.getStatus(), streamLoadItem.getMessage(), streamLoadItem.getUrl(), streamLoadItem.getTotalRows(), streamLoadItem.getLoadedRows(),
+                                    streamLoadItem.getFilteredRows(), streamLoadItem.getUnselectedRows(), streamLoadItem.getLoadBytes(), streamLoadItem.getStartTime(),
+                                    streamLoadItem.getFinishTime());
+
+                            AuditEvent auditEvent = new StreamLoadAuditEvent.AuditEventBuilder().setEventType(EventType.STREAM_LOAD_FINISH)
+                                    .setLabel(streamLoadItem.getLabel()).setDb(streamLoadItem.getDb()).setTable(streamLoadItem.getTbl())
+                                    .setUser(streamLoadItem.getUser()).setClientIp(streamLoadItem.getUserIp()).setStatus(streamLoadItem.getStatus())
+                                    .setMessage(streamLoadItem.getMessage()).setUrl(streamLoadItem.getUrl()).setTotalRows(streamLoadItem.getTotalRows())
+                                    .setLoadedRows( streamLoadItem.getLoadedRows()).setFilteredRows(streamLoadItem.getFilteredRows())
+                                    .setUnselectedRows(streamLoadItem.getUnselectedRows()).setLoadBytes(streamLoadItem.getLoadBytes())
+                                    .setStartTime(streamLoadItem.getStartTime()).setFinishTime(streamLoadItem.getFinishTime())
+                                    .build();
+                            Catalog.getCurrentCatalog().getAuditEventProcessor().handleAuditEvent(auditEvent);
+                            if (entry.getKey().compareTo(backend.getLastStreamLoadTime()) > 0) {
+                                backend.setLastStreamLoadTime(entry.getKey());
+                            }
+                        }
+                        ok = true;
+                    } catch (Exception e) {
+                        LOG.warn("task exec error. backend[{}]", backend.getId(), e);
+                    } finally {
+                        if (ok) {
+                            ClientPool.backendPool.returnObject(address, client);
+                        } else {
+                            ClientPool.backendPool.invalidateObject(address, client);
+                        }
+                    }
+                }
+                LOG.info("finished to pull stream load records of all backends. cost: {} ms",
```

Comment:
You can add the number of records in this log

Reply:
> You can add the number of records in this log

OK.